.github/CODEOWNERS

```diff
@@ -195,7 +195,7 @@ receiver/kafkareceiver/                                  @open-telemetry/collect
 receiver/kubeletstatsreceiver/                           @open-telemetry/collector-contrib-approvers @dmitryax
 receiver/memcachedreceiver/                              @open-telemetry/collector-contrib-approvers @djaglowski
 receiver/mongodbreceiver/                                @open-telemetry/collector-contrib-approvers @djaglowski @schmikei
-receiver/mongodbatlasreceiver/                           @open-telemetry/collector-contrib-approvers @zenmoto
+receiver/mongodbatlasreceiver/                           @open-telemetry/collector-contrib-approvers
```

Comment:
I think we should remove this line entirely, put the component [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/dc2f6e135d5a157b6b0537b8721d80dc1bcc6fa7/.github/ALLOWLIST#L25) instead and open an issue like [these ones](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=label%3Aunmaintained+is%3Aclosed)

Reply:
I was going to sweep through the whole file next, if that’s ok.